src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -146,6 +147,23 @@ public SubscriptionController(final SubscriptionDbRepository subscriptionReposit
         }
     }
 
+    @RequestMapping(value = "/{id}/stats", method = RequestMethod.GET)
+    public ResponseEntity<?> getSubscriptionStats(@PathVariable("id") final String subscriptionId,
+                                             final NativeWebRequest request) {
+        try {
+            final Subscription subscription = subscriptionRepository.getSubscription(subscriptionId);
+            final List<SubscriptionEventTypeStats> subscriptionStat =
+                    subscriptionService.createSubscriptionStat(subscription);
+            return status(OK).body(new ItemsWrapper(subscriptionStat));
+        } catch (final NoSuchSubscriptionException e) {
+            LOG.debug("Failed to find subscription: " + subscriptionId, e);
```

Comment:
Maybe it's better to use `LOG.debug("Failed to find subscription: {}", subscriptionId, e);` to avoid unnecessary string concatenations.